package.json

```diff
@@ -69,6 +70,13 @@
     "test": "node --max_old_space_size=4096 ./node_modules/.bin/ckeditor5-dev-tests --reporter=dots",
     "install-optional-dependencies": "./scripts/install-optional-dependencies.sh",
     "switch-to-dev-dev": "./scripts/switch-to-dev-dev.sh",
-    "build-and-publish-docs": "./scripts/docs/build-and-publish.js"
+    "build-and-publish-docs": "./scripts/docs/build-and-publish.js",
+    "lint": "eslint --quiet '**/*.js'",
```

Comment:
Please move these to the top. They are more important than the other tasks which are our super-dev tasks, in fact.